ipynb/android/workloads/Android_Recents_Fling.ipynb

```diff
@@ -329,22 +348,26 @@
     "    # Reset framestats collection\n",
     "    target.execute('dumpsys gfxinfo --reset')\n",
     "    \n",
+    "    w, h = target.screen_resolution\n",
+    "    x = w/2\n",
+    "    yl = int(0.1*h)\n",
```

Comment:
Had to change this to `0.2*h` on the N5X, otherwise it touches the google search bar and doesn't fling...